planet/src/main/java/com/google/planet/servlets/SendEmailServlet.java

```diff
@@ -34,47 +35,60 @@
+                message.setFrom(new InternetAddress("alicexyz@google.com", "Alice Zhou"));
+                message.addRecipient(Message.RecipientType.TO,
+                                new InternetAddress(userEmail));
+                message.setSubject("Your itinerary for " + listName);
+                Multipart multipartContent = new MimeMultipart();
 
-            // Add a pdf attachment of the itinerary
-            // TODO: change input to actual itinerary
-            MimeBodyPart attachment = new MimeBodyPart();
-            String itinerary = "your itinerary";
-            InputStream attachmentDataStream = new ByteArrayInputStream(itinerary.getBytes("UTF-16"));
-            attachment.setFileName("itinerary.pdf");
-            attachment.setContent(attachmentDataStream, "application/pdf");
-            multipartContent.addBodyPart(attachment);
+                // Add email body
+                MimeBodyPart htmlPart = new MimeBodyPart();
+                String htmlBody = "<div>Hello! <h3> Thank you for using Planet.</h3> " +
+                    "<h3>You can see a copy of your itinerary in the attachment below.</h3></div>";  
+                htmlPart.setContent(htmlBody, "text/html");
+                multipartContent.addBodyPart(htmlPart);
 
-            message.setContent(multipartContent);
-            Transport.send(message);
-        } catch (MessagingException e) {
-            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-            response.getWriter().println(e.getMessage());
-        } catch (UnsupportedEncodingException e) {
-            response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-            response.getWriter().println(e.getMessage());
-        } 
+                // Add a pdf attachment of the itinerary
+                MimeBodyPart attachment = new MimeBodyPart();
+                InputStream attachmentDataStream = new ByteArrayInputStream(itineraryContent.getBytes("UTF-16"));
+                attachment.setContent(attachmentDataStream, "text/plain");
+                Date date= new Date();
+                long time = date.getTime();
+                String fileName = "itinerary" + (Long.toString(time)) +".txt";
+                attachment.setFileName(fileName);
+                multipartContent.addBodyPart(attachment);
+
+                message.setContent(multipartContent);
+                Transport.send(message);
+            } catch (MessagingException e) {
+                response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                response.getWriter().println(e.getMessage());
+            } catch (UnsupportedEncodingException e) {
+                response.setStatus(HttpServletResponse.SC_BAD_REQUEST);
+                response.getWriter().println(e.getMessage());
+            } 
+        } else {
+            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
+            response.getWriter().println("");
```

Comment:
Is this necessary?

Reply:
Not really... I just removed it, thank you!